app/controllers/users_controller.rb

```diff
@@ -5,7 +5,7 @@ def new
 
   def create
     if session[:user]
-      flash[:notice] = 'Please log out before creating a new user'
+      flash[:alert] = 'Please log out before creating a new user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.